src/main/java/de/retest/recheck/ui/descriptors/SutState.java

```diff
@@ -22,25 +25,42 @@
 	@XmlElement
 	private final List<RootElement> descriptors;
 
+	@XmlElement
+	private final Map<String, String> metadata;
+
 	@SuppressWarnings( "unused" )
 	private SutState() {
 		super( PERSISTENCE_VERSION );
 		// for JAXB
 		descriptors = new ArrayList<>();
+		metadata = new HashMap<>();
 	}
 
 	public SutState( final Collection<RootElement> set ) {
+		this( set, new HashMap<>() );
```

Comment:
I would always prefer `Collections.emptyMap()` for such cases to avoid creating multiple instances.